steering_controllers/tricycle_controller/CMakeLists.txt

```diff
@@ -6,26 +6,34 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif()
 
 # find dependencies
```

Comment:
The same as above, no need for a separate package.

Reply:
I'm keeping this package ATM to help me with the exact implementation of the tricycle controller. I will delete it once I c/p stuff for odometry if they are needed.